src/gui/transaction.go

```diff
@@ -160,7 +181,7 @@ func getTransactions(gateway Gatewayer) http.HandlerFunc {
 		}
 
 		// Converts visor.Transaction to visor.TransactionResult
-		txRlts, err := visor.NewTransactionResults(txns)
```

Comment:
`visor.NewTransactionResults(txns)` is in rpc.go. A new version was created at the end of this file (modified to include the inputs data) to mantain it separated from the rpc code.